### ISSUE TITLE
Notification of language requests lists wrong user

### ISSUE THREAD

**Issue by rakekniven** [rakekniven/Transifex-Issues#12]
Today I have seen two language requests for Nextcloud projects.

Notification looks as follow:
![grafik](https://user-images.githubusercontent.com/2069590/30972675-89bc98f0-a46b-11e7-8f1e-43f55694baa3.png)

But user only requested one language. Timeline shows following:
![grafik](https://user-images.githubusercontent.com/2069590/30972719-aff08716-a46b-11e7-8471-9ccb761bf3e2.png)

**Comment by rakekniven** [comment]
Got an answer from Transifex:

> Hi rakekniven,
> 
> Thank you for reaching out to us and I am sorry for the delay in getting back to you.
> 
> __We will try to answer all of your questions as soon as possible.
> 
> We really appreciate you taking the time to share your feedback with us.
> 
> Regards,
> Nxxx
> 
> --
> Nxxx Exxx
> Customer Success Engineer | Transifex

**Comment by comradekingu** [comment]
That is not an answer.

**Comment by rakekniven** [comment]
Happened again

![image](https://user-images.githubusercontent.com/2069590/31724496-2c20f756-b422-11e7-818b-754e46f67340.png)

![image](https://user-images.githubusercontent.com/2069590/31724482-1c27244c-b422-11e7-99f9-97cde0e4fafa.png)

@samuelchan22 Could you please check this one?

**Comment by samuelchan22** [comment]
@rakekniven For some of our notifications, we group them so we don't spam you or other users. The idea is similar to Facebook notifications. In this case, the wording for the in-app notification creates the impression that Htike requested both Burmese and Spanish (Colombia). But in reality, Htike and several others requested those languages. We're going to fix the wording so it's clearer.

**Comment by rakekniven** [comment]
> We're going to fix the wording so it's clearer.

Hello @samuelchan22 Thank you for digging into it. When could we expect to see a change?

**Comment by rakekniven** [comment]
@samuelchan22 Any update?

**Comment by samuelchan22** [comment]
@rakekniven Sorry for the delay, we've updated the notification to say the following:

> The Greek language and 1 other languages were requested for the Transifex project

We decided to drop the requester names from the notification since 1) there wasn't an easy way to make things clear without the notification being really long and 2) they didn't add much value.

**Comment by rakekniven** [comment]
Hello @samuelchan22 Yesterday I had a request for french language. There is french already, but different locale value. 
So how to contact the requesting person and inform it about already existing language and forward him?

Solution: I had to search the timeline to find the person.

So IMHO it is bad to drop the name. Can you follow my argumentation?

**Comment by samuelchan22** [comment]
@rakekniven I see what you're saying. We're going to see how we can fix this. Most likely, we'll still keep the notification simple and have the details exposed in the project timeline. Otherwise, if you get multiple language requests from multiple people, the notification will become hard to follow.

**Comment by rakekniven** [comment]
Hello @samuelchan22 
Nextcloud is an active project. But not as active as there are language requests daily.

My solution would be:

- Keep the name in the notification
- Do not combine two requests into one notification

Another issue related here:
90% of the requests are duplicates. 
1. User is requesting e.g. French (fr_FR) then discovers there is French already (FR). The user joins existing French team and I have to decline the request
2. User is requesting e.g. French (fr_FR) and does NOT discover there is French already (FR). I sent a message to the user and pointing him to the French team and decline the request.
So I wonder why there are so many "bad" requests? IMHO Transifex should check the process/workflow/usability of requesting languages.
Should I file a separate issue?

**Comment by samuelchan22** [comment]
@rakekniven Thanks for the feedback. For the time being, we decided to keep the notifications grouped and without the username because projects that are more active might get more requests and we want to avoid the in-app notification feed from being filled with these requests. You'll still be able to see the username in the project timeline. 

As for the other issue you mentioned, it makes a ton of sense. I'd say open a ticket for it as it's a related but separate issue. Maybe if we fix that we can revisit this notification.

**Comment by rakekniven** [comment]
Hello @samuelchan22 . Sad to hear that. I filed a bug and now I need more clicks to get the person filing the request...
